frontend/src/SignUp.jsx

```diff
@@ -0,0 +1,71 @@
+import InputBox from "./components/InputBox";
+import { useNavigate } from "react-router-dom";
+
+const SignUp = () => {
+  const navigate = useNavigate();
+
+  return (
+    <div className="flex flex-col">
+      <header>
+        <div className="text-center text-8xl m-20 font-bold text-indigo-50 text-shadow-lg">
+          Alpha-Edge
+        </div>
+      </header>
+      <form className="flex flex-col bg-indigo-50 p-8 rounded-md shadow-xl/40 shadow-slate-900 w-150 m-auto">
+        <h3 className="font-bold text-3xl p-1 text-center">Sign Up</h3>
+        <p className="font-bold text-sm p-3 text-center">
+          Please sign up to continue{" "}
+        </p>
+        <InputBox
+          placeholder={"ex: Santiago Criado"}
+          label={"Name"}
+          id={"name"}
+        />
+        <InputBox
+          placeholder={"ex: 1234"}
+          label={"Set Password"}
+          id={"new-password"}
+        />
+        <InputBox
+          placeholder={"ex: Cool-Dude-1"}
+          label={"Username"}
+          id={"name"}
+        />
+        <InputBox
+          placeholder={"bobby@gmail.com"}
+          label={"email"}
+          id={"email"}
+        />
+        <button
+          id="signup"
+          className="m-2 bg-green-400 shadow-xl/10 shadow-slate-900"
+        >
+          Sign-Up
+        </button>
+        <button
+          onClick={() => {
+            navigate("/");
+          }}
+          id="login"
+          className="m-2 bg-fuchsia-950 text-white shadow-xl/10 shadow-slate-900"
+        >
+          back to login
```

Comment:
"Back to login". Keep correct word capitalization. [Sentence case](https://editorninja.com/wp-content/uploads/2023/04/Sentence-Case-Vs.-Title-Case-%E2%80%94-Whats-The-Difference.webp) is advised for most of the strings within your app.